app/sections/about/about.component.ts

```diff
@@ -18,53 +18,33 @@ import {transformResponse} from '../../shared/tools/response.tools';
   pipes: [MarkdownPipe]
 })
 export class About implements OnActivate {
-  private content: any;
-  private submenuItems: any[] = [];
+  private content: NodePageContent;
+  private submenuItems: NodePageContent[] = [];
 
   constructor(private _routerData: RouteData,
               private _params: RouteParams,
-              private _contentful: ContentfulService) {
+              private _contentfulContent: ContenfulContent) {
   }
 
   routerOnActivate(to: ComponentInstruction, from: ComponentInstruction): any {
     return new Promise(
       (resolve) => {
         let slug = this._params.get('slug') || this._routerData.get('contentfulSlug');
-        this._contentful
-          .withLinksLevel(2)
-          .getEntryBySlug(
-            ContentfulConfig.CONTENTFUL_NODE_PAGE_TYPE_ID,
-            slug
-          )
-          .map(response => response.json())
+        this._contentfulContent
+          .getAboutPage(slug)
           .subscribe(
             response => {
-              this.submenuItems = this.getSubmenuFromResponse(response);
-              this.content = transformResponse<any>(response)[0];
+              this.submenuItems = response.submenuItems;
+              this.content = response.content;
               resolve(true);
             },
-            error => console.log(error)
+            error => {
+              // what we should do here ?
```

Comment:
for example in here we can create notifications regarding errors using https://www.npmjs.com/package/toastr
But this is thing that should be approved by Ola and designers.

Or we can grab some placeholders and render them instead of a real content